### PR TITLE
Add global accelerator to AWS policy list

### PR DESCRIPTION
**Pull request by ianarsenault**
Updating the docs to include global accelerator as part of our AWS policy list needed for Snowplow Admin role

### CODE REVIEW

**Review by stanch — APPROVED** [review]
LGTM but I have no clue if this is correct or not, for that I defer to @Andy-Hay

**Comment by Andy-Hay** [comment]
LGTM as well :)

**Review by Andy-Hay — APPROVED** [review]
LGTM

**Pull request merged**
